tests/playbooks/tests_infiniband.yml

```diff
@@ -0,0 +1,160 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - name: "INIT Prepare setup"
+      debug:
+        msg: "##################################################"
+    - block:
+        # For initscripts provider, the script
+        # `/etc/sysconfig/network-scripts/ifup-ib` is required in order to
+        # configure the infiniband profiles
+        - name: Validate the script 'ifup-ib' existed
+          command: test -x /etc/sysconfig/network-scripts/ifup-ib
+          register: ifup_ib_exist
+          failed_when: ifup_ib_exist.rc == 1
+          when: network_provider == "initscripts"
```

Comment:
Where does this script come from?

Reply:
https://raw.githubusercontent.com/Mellanox/rdma-utils/master/rdma.ifup-ib

Actually it was included in rhel-7.9 and dropped in rhel-8.0. 
https://pkgs.devel.redhat.com/cgit/rpms/rdma-core/commit/?h=rhel-7.9&id=697c706324f2c866d7b855682a8a3c80371a6651
https://pkgs.devel.redhat.com/cgit/rpms/rdma-core/commit/?h=rhel-8.0&id=1fbba1120742c5446ffeed9cb8bb91542b8ea317

Reply:
I will address this.

Reply:
Yes. I can limit the test for initscripts provider in only RHEL 7. But there is a possibility that copying the `/etc/sysconfig/network-scripts/ifup-ib` from the RHEL-7 machine to other RHEL os version machine prior to the testing.